x-pack/platform/plugins/private/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -107,7 +108,7 @@ export const TransformManagement: FC = () => {
   const styles = useStyles();
   const [transformPageState, setTransformPageState] = usePageUrlState<PageUrlState>(
     'transform',
-    getDefaultTransformListState()
+    useUrlState('_a')?.[0] || getDefaultTransformListState()
```

Comment:
we don't need these changes here at all + these changes override the default state.

So please undo these changes, and here https://github.com/elastic/kibana/pull/237177/files#diff-0451906fb7423eee016b8c90d5f45856251a2e8317e7734763e7ba0b42899a28R15-R17

just make sure to wrap your page state when transitioning into `transform` key on your side, the mechanism of extracting page state and merging into default page state is already handled by `usePageUrlState` automatically

i.e. 

```tsx
kbnRison.encode({
    transform: {
      queryText: transformId,
     }
  });
```